packages/apt/build.sh

```diff
@@ -19,8 +18,9 @@ etc/apt/trusted.gpg
 "
 
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--DPERL_EXECUTABLE=$(which perl)
+-DPERL_EXECUTABLE=$(command -v perl)
 -DCMAKE_INSTALL_FULL_LOCALSTATEDIR=$TERMUX_PREFIX
+-DCACHE_DIR=/data/data/com.termux/cache
```

Comment:
This is currently calculated from the above variable to be `$TERMUX_PREFIX/var/cache/apt`, can change it to `/data/data/com.termux/cache/apt` if further scoping is wanted.

Reply:
@buttaface better to use a subdirectory here instead of cachedir root.

Reply:
Done.